src/frontend-scripts/components/section-main/Game.jsx

```diff
@@ -97,7 +97,7 @@ export default class Game extends React.Component {
 		const { userInfo, gameInfo, socket } = this.props;
 		let isFlappy;
 
-		isFlappy = true;
+		isFlappy = false;

```

Comment:
Let's leave this unchanged in PRs to keep the dev env consistent and avoid future conflicts.

Reply:
Sorry! I forgot to revert that.

Reply:
Not a big deal, I'll fix prior to merge. Future reference, you can just be very specific on what files you stage in your commits to only stage the changes you want in your commits.

Reply:
Yes, I know. Thank you.